src/apiCalls.tsx

```diff
@@ -28,7 +28,7 @@ export const postAWedding = (wedding: any) => {
   })
   .then(response => {
     if(response.ok) {
-      return response
+      return response.json()
```

Comment:
Was this part of what was leading to some bugs?

Reply:
It was part of it for sure. We were originally adding the newest wedding to the weddings on the dashboard using a date.now id, so when we clicked on the wedding it was getting an error because the id didn't exist in the database until the page was refreshed and the date.now id was replaced by the id created in the backend.

Reply:
Thanks for the clarification!